### PR TITLE
UXUI-235 moving all components into the ui-toolkit itself

### PR DESCRIPTION
**Pull request by jackdcrawford**
#### What does this PR do?
Initial move of all components that are currently living in their own repositories into the `ui-toolkit`.

Components moved into `src/components/` directory:
[button](https://github.com/holidayextras/ui-component-button)
[flag](https://github.com/holidayextras/ui-component-flag)
[rating](https://github.com/holidayextras/ui-component-rating)
[reviews](https://github.com/holidayextras/ui-component-reviews)
[tile](https://github.com/holidayextras/ui-component-tile)

Once this is complete, we will start removing the other repositories & streamlining the workflow within ui-toolkit of actually working on the components in the browser.

This also removes our namespaced Bootstrap export

#### What unit or integration tests does this PR have?
N/A - each component already has it's own test

#### What selenium tests does this PR have?
N/A

#### How should a developer review this?
This PR isn't as big as it looks, the main changes are to the README & the components being referenced in `src/ui-toolkit.js`, these would be good places to start.

#### How should this be manually tested?
Follow the [updated README](https://github.com/holidayextras/ui-toolkit/blob/ea5ed6c781063f8d5023583035e0b903d54ab320/README.md#adding-a-new-component) for how to add a new component & view the docs.

#### What are the relevant tickets?
[UXUI-235](https://holidayextras.jira.com/browse/UXUI-235)

#### Screenshots (if appropriate)
N/A

#### What gif best describes how you feel about this work?
eggs in one basket

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've checked there is appropriate selenium test coverage.
- [x] I've included regression tests for any bug fixes.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this on mobile Android, Safari, Chrome for any visual changes.
- [x] I've tested this on desktop IE9/10/11, Firefox, Chrome for any visual changes.
- [x] I've checked any significant visual changes with a member of the UX/UI pod.
- [x] I've checked this work against the requirements of the Jira.
- [x] I've added and updated translations for all supported languages
- [x] I've informed those who need to know of any unforeseen impact of this work (database migrations, deployment procedure changes).
- [x] I am ready for this to be code reviewed, merged and tested.
- [x] I've thought about writing a blog post about this

#### IP or Developer Review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] +1 from me!

#### Software Engineer or Developer review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] +1 from me!

#### Software Engineer or project guru review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by Joezo** [comment]
i'll take the dev slot on this one

**Comment by jodiedoubleday** [comment]
Take SE

**Comment by jodiedoubleday** [comment]
:+1: looks good for a first step, plenty of improvements we can do in the future to this :smiling_imp:

**Comment by HarleyRowland** [comment]
Taking IP review

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/holidayextras/ui-toolkit.git"
   },
   "scripts": {
-    "postinstall": "cd docs; npm install; cd -"
+    "postinstall": "./scripts/subinstall.sh"
```
Comment (Joezo) [inline package.json]: can we call the script `postinstall.sh` just to be clearer

File: src/components/button/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# ui-component-button
+Our main button component.
+
+## Installation
+To install this module:
+
+`npm install [git repo]`
```
Comment (Joezo) [inline src/components/button/README.md]: not `git repo` anymore

File: src/components/button/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+## Building a distributable version
+To build a distributable version of the component:
+
+`npm run build-dist`
+
+## Parameters
+
+## Example
+
+```
```
Comment (Joezo) [inline src/components/button/README.md]: add `jsx`? :p

**Comment by HarleyRowland** [comment]
+1

File: src/components/button/README.md (hunk 2 above)
Comment (jackdcrawford) [inline src/components/button/README.md]: Planning to address this when we make the components **cherrypickable** < new word

**Comment by Joezo** [comment]
:+1: :D

**Pull request merged**